assistant/src/main/java/com/ibm/watson/assistant/v2/model/MessageContextSkill.java

```diff
@@ -25,12 +25,12 @@
   @SerializedName("user_defined")
   protected Map<String, Object> userDefined;
 
-  protected Map<String, Object> system;
+  protected MessageContextSkillSystem system;
```

Comment:
Need to revert his, handwritten changes needed.

Reply:
fixed in [c434197d28d928f32c618f35f2bd67a2780dc00b](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/c434197d28d928f32c618f35f2bd67a2780dc00b)